### PR TITLE
Feature/migrate to js

### PR DESCRIPTION
**Pull request by ceseale**
closes: https://github.com/polyswarm/contracts/issues/43

### CODE REVIEW

**Comment by myhndl** [comment]
bump @scsich

File: scripts/safe_migrate.js [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+	  `/BountyRegistry`,
+	  `/ERC20Relay`,
+	  `/NectarToken`,
+	  `/OfferLib`,
+	  `/OfferMultiSig`,
+	  `/OfferRegistry`
+	];
+	let missingABI = false;
+	let consulBaseUrl = `chain/${args['poly-sidechain-name']}`;
+	let respone;
```
Comment (Rizato) [inline scripts/safe_migrate.js]: Looks like a typo of response to me.

File: scripts/safe_migrate.js [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+	for (let path of paths) {
```
Comment (Rizato) [inline scripts/safe_migrate.js]: New syntax for me, good to know.

**Comment by myhndl** [comment]
@Rizato said he'd review this.

File: scripts/safe_migrate.js (hunk 1 above)
Comment (ceseale) [reply scripts/safe_migrate.js]: whoops thanks!

**Comment by ceseale** [comment]
@Rizato It's still using [create_and_mirgrate.sh](https://github.com/polyswarm/contracts/pull/45/files?utf8=%E2%9C%93&diff=unified#diff-fe35a48e7a2311e66a130749e66391d5). I've replaced all that bash code with safe_mirgrate.js for easier error handling. Most of [create_config.js](https://github.com/polyswarm/contracts/blob/f6d6ad581cd78f972e0130323a18220de3eb76a7/scripts/create_config.js#L120-L127) is the same so it still puts the contracts in consul.

File: scripts/safe_migrate.js (hunk 2 above)
Comment (ceseale) [reply scripts/safe_migrate.js]: :-1:

**Pull request merged**